src/SeaNetMicron.cpp

```diff
@@ -89,43 +98,35 @@ void Micron::configure(const MicronConfig &config,uint32_t timeout)
     //some conversions
     int ad_interval = (((config.resolution/config.speed_of_sound)*2.0)*1e9)/640.0;
     int number_of_bins = config.max_distance/config.resolution;
-    uint16_t left_limit = (((M_PI-config.left_limit.rad)/(M_PI*2.0))*6399.0);
-    uint16_t right_limit = (((M_PI-config.right_limit.rad)/(M_PI*2.0))*6399.0);
+    uint16_t left_limit = (((M_PI+config.left_limit.rad)/(M_PI*2.0))*6399.0);
+    uint16_t right_limit = (((M_PI+config.right_limit.rad)/(M_PI*2.0))*6399.0);
```

Comment:
Thanks for the info about the micron. Just to be sure: I remember that the micron's zero is at the back. Meaning that zero is the back of the device, and 6399 is one step counter-clockwise from the back. Is that right ?